addons/api/addon/initializers/fragment-string.js

```diff
@@ -0,0 +1,12 @@
+// app/initializers/fragment-serializer.js
```

Comment:
This comment shows where the serializer lives?

Reply:
This is default generated.